src/layouts/ultimate-guide.tsx

```diff
@@ -101,7 +105,7 @@ const UltimateGuideLayout: FunctionComponent<LayoutProps> = ({
           </header>
 
           <main className="prose md:prose-lg max-w-none">
-            <div>{children}</div>
+            <MDXProvider components={components}>{children}</MDXProvider>
```

Comment:
suggestion: pass mdx components [higher up in the tree](https://github.com/eggheadio/egghead-next/blob/main/src/pages/_app.tsx#L46) so that all layouts can use it.